XXX Relax CMake to not require Qt6 AND Qt5, stick to Qt5
relevant issue:    https://github.com/open-eid/DigiDoc4-Client/issues/1097
qt6 build failure: https://github.com/open-eid/DigiDoc4-Client/pull/1098

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -14,7 +14,9 @@ include( VersionInfo )
 find_package( PKCS11 )
 find_package( LibDigiDocpp 3.14.8 REQUIRED )
 find_package( LDAP REQUIRED )
-find_package(QT NAMES Qt6 Qt5 COMPONENTS Core REQUIRED)
+find_package(Qt5 COMPONENTS Core REQUIRED)
+set(QT_VERSION ${Qt5_VERSION})
+set(QT_VERSION_MAJOR ${Qt5_VERSION_MAJOR})
 find_package(Qt${QT_VERSION_MAJOR} 5.9.0 REQUIRED COMPONENTS Core Widgets Network PrintSupport Svg LinguistTools)
 
 set_env( TSL_URL "https://ec.europa.eu/tools/lotl/eu-lotl.xml" CACHE STRING "TSL trust list primary URL" )
